kraft.yaml

```diff
@@ -10,7 +10,7 @@ unikraft:
     - CONFIG_LIBVFSCORE_AUTOMOUNT_ROOTFS=y
     - CONFIG_LIBVFSCORE_ROOTFS_9PFS=y
     - CONFIG_LIBUKLIBPARAM=y
-    - CONFIG_LIBVFSCORE_ROOTDEV="fs0"
```

Comment:
@mogasergiu 

Is this change necessary? I think it's better to left with fs0 because otherwise it won't work using kraft.
Kraft is using qemu-guest to start the application and qemu-guest have support only for tag as `fs0`, `fs1`, etc.
Otherwise it would need an extra step adding, to change in `menuconfig` from `rootfs`  to `fs0`.